COPY/usr/lib/systemd/system/evmserverd.service

```diff
@@ -6,6 +6,7 @@ After=syslog.target
 ExecStart=/bin/sh -c "/bin/evmserver.sh start"
 ExecStop=/bin/sh -c "/bin/evmserver.sh stop"
 Type=forking
+Restart=on-failure
```

Comment:
@jrafanie Yeah, it will bring the service back up when it exits with any non-zero value.